account/wallet.go

```diff
@@ -0,0 +1,25 @@
+package account
+
+import (
+	"crypto/ecdsa"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+func CreateAddress() (address, pubKey, privKey string, err error) {
+	key, err := crypto.GenerateKey()
+	if err != nil {
+		return "", "", "", err
+	}
+
+	privKeyBytes := crypto.FromECDSA(key)
+	pubKeyBytes := crypto.FromECDSAPub(key.Public().(*ecdsa.PublicKey))
+	addressBytes := crypto.PubkeyToAddress(*key.Public().(*ecdsa.PublicKey))
```

Comment:
@zachzundel if you want you can just use:
```
	pubKeyBytes := crypto.FromECDSAPub(&key.PublicKey)
	addressBytes := crypto.PubkeyToAddress(key.PublicKey)
```

Reply:
Thanks!